src/commands/install.yml

```diff
@@ -16,7 +20,7 @@ parameters:
     default: 'latest'
   no-output-timeout:
     description: >
-      Elapsed time the tests can run without output. The string is a decimal with unit suffix,
+      Elapsed time the install can run without output. The string is a decimal with unit suffix,
```

Comment:
What does "the install" mean here? Shouldn't we use a noun (installation, install step, ...)?

Reply:
"install command"?

Reply:
updated